superset/views/database/mixins.py

```diff
@@ -197,7 +195,7 @@ class DatabaseMixin:
         "extra": _("Extra"),
         "encrypted_extra": _("Secure Extra"),
         "server_cert": _("Root certificate"),
-        "allow_run_async": _("Asynchronous Query Execution"),
+        "allow_run_async": _("Async Execution"),
```

Comment:
It appears this is translated in the ZH language pack: https://github.com/apache/incubator-superset/blob/c716f7afe8f528281950dea24f01c245690cad9e/superset/translations/zh/LC_MESSAGES/messages.po#L4031-L4032

Would be nice to change the reference so that the translation doesn't break.

Reply:
I think the `#` here means that this was actually commented out at one point? I think there's something not set up quite right with the language packs... But there should be a script to run to pull in this new string into the translation packs. And then we could move the old string to the new one

Reply:
Yeah we have bigger issues on `i18n`, the main one being that `superset-ui` doesn't get "inventorize" when scanning for new strings.

I tend to think of `i18n` as another parallel process that's not coupled to dev work. Technically we should re-publish the strings periodically, but not as part of everyday PRs